ppa/editorial/tests.py

```diff
@@ -89,9 +90,23 @@ def test_routing(self):
         response = self.client.get(editorial_index_url)
         assert response.status_code == 200
         # basic template/content check
+        # NOTE: Should these be refactored into separate template
+        # checking test?
         self.assertContains(response, index_page.title)
         self.assertContains(response, editorial_page.title)
         self.assertContains(response, editorial_page.relative_url(site))
+        # each should be shown once
+        person_a = Person.objects.get(name='Person A')
+        person_b = Person.objects.get(name='Person B')
+        self.assertContains(response, person_a.name, count=1)
+        self.assertContains(response, person_b.name, count=1)
```

Comment:
Maybe check that "By " is only displayed once, as part of fixing that logic for multiple authors.